### PR TITLE
JOBS-1044: add jobs-api last completion dates per locale

### PR DESCRIPTION
**Pull request by andreyfurdylo**

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -5616,6 +5616,53 @@ paths:
                     details: null
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
+  /jobs-api/v3/projects/{projectId}/jobs/{translationJobUid}/locales-completion-dates:
+    get:
+      summary: "Get last completion dates per locale of job"
+      tags:
+      - Jobs
+      description: "This get last completion dates per locale of job"
```
Comment (DmitryBochorishvili) [inline spec/swagger.yaml]: Does the endpoint support `closed`/`deleted` jobs? Do we need to indicate such information in api documentation?

Comment (andreyfurdylo) [reply spec/swagger.yaml]: yes, we do not have any limitation. Even locale was deleted from a job, this endpoint will return information for this locale.

File: spec/swagger.yaml [hunk 2]
```diff
@@ -5616,6 +5616,53 @@ paths:
                     details: null
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
+  /jobs-api/v3/projects/{projectId}/jobs/{translationJobUid}/locales-completion-dates:
+    get:
+      summary: "Get last completion dates per locale of job"
+      tags:
+      - Jobs
+      description: "This get last completion dates per locale of job for all locales that were ever completed."
```
Comment (jones-smartling) [inline spec/swagger.yaml]: i would put description as "Returns the last completion date for any locale completed within the job."

File: spec/swagger.yaml [hunk 3]
```diff
@@ -5616,6 +5616,53 @@ paths:
...
+      consumes:
+      - application/json
+      parameters:
+      - name: projectId
```
Comment (jones-smartling) [inline spec/swagger.yaml]: parameters should have a description

**Pull request merged**